manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
```

```suggestion
PackageVersion: 22.5.15
```
This must match Control Panel.